src/SDKs/SqlManagement/Sql.Tests/Utilities/TestEnvironmentUtilities.cs

```diff
@@ -9,6 +9,75 @@ namespace Sql.Tests
 {
     public static class TestEnvironmentUtilities
     {
+        private const string _environmentVariableName = "TEST_CSM_ORGID_AUTHENTICATION";
+
+        private static readonly TestEnvironment _environment = 
```

Comment:
@nathannfan any reason you cannot have single instance of TestEnvironment across your entire stack that you can reuse.
You already create an instance while creating Clients, why not reuse the same instance, rather than creating a new one. Not creating a new one will allow you to move away from the dependency on "TEST_CSM_ORGID_AUTHENTICATION", this variable is exclusively used inside TestFramework and no guarantee this will not change in the future as TestFramework evolves